botnim/cli.py

```diff
@@ -53,68 +53,65 @@ def reverse_lines(text: str) -> str:
     return "\n".join(reversed_lines)
 
 @query_group.command(name='search')
-@click.argument('environment', type=click.Choice(['production', 'staging']))
-@click.argument('bot', type=click.Choice(AVAILABLE_BOTS), default='takanon')
+@click.argument('environment', type=click.Choice(VALID_ENVIRONMENTS))
+@click.argument('bot', type=click.STRING)
 @click.argument('context', type=click.STRING)
-@click.argument('query_text', type=str)
+@click.argument('query_text', type=click.STRING)
 @click.option('--num-results', type=int, default=7, help='Number of results to return')
 @click.option('--full', '-f', is_flag=True, help='Show full content of results')
 @click.option('--rtl', is_flag=True, help='Display results in right-to-left order')
 def search(environment: str, bot: str, context: str, query_text: str, num_results: int, full: bool, rtl: bool):
     """Search the vector store with the given query."""
     try:
-        search_results = run_query(query_text, environment, bot, context, num_results)
+        search_results = run_query(environment, bot, context, query_text, num_results, format="dict")
         for result in search_results:
             formatted_result = format_result(result, show_full=full)
             if rtl:
                 formatted_result = reverse_lines(mirror_brackets(formatted_result))
             click.echo(formatted_result)
     except Exception as e:
         click.echo(f"Error: {str(e)}", err=True)
-        raise click.Abort()
 
 @query_group.command(name='list-indexes')
-@click.argument('environment', type=click.Choice(['production', 'staging']))
-@click.option('--bot', type=click.Choice(AVAILABLE_BOTS), default='takanon', 
-              help='Bot to list indexes for')
+@click.argument('environment', type=click.Choice(VALID_ENVIRONMENTS))
+@click.option('--bot', type=click.STRING, default=None, help='Filter indexes by bot name')
```

Comment:
why?

Reply:
oh, I did not consider well the meaning of this change. fixed.